### PR TITLE
Fixed videos with no adaptive streams

### PR DESCRIPTION
**Pull request by swiftyy-mage**
https://github.com/nficano/pytube/issues/577

### CODE REVIEW

File: pytube/extract.py [hunk 1]
```diff
@@ -277,13 +277,20 @@ def apply_descrambler(stream_data: Dict, key: str) -> None:
...
     if key == "url_encoded_fmt_stream_map" and not stream_data.get(
         "url_encoded_fmt_stream_map"
-    ):
-        formats = json.loads(stream_data["player_response"])["streamingData"]["formats"]
-        formats.extend(
-            json.loads(stream_data["player_response"])["streamingData"][
-                "adaptiveFormats"
+    ):        
+        try:
+            formats = json.loads(stream_data["player_response"])["streamingData"][
```
Comment (hbmartin) [inline pytube/extract.py]: Looks like this can be lifted out of the try/except block?

**Comment by swiftyy-mage** [comment]
Will work on this now.

**Comment by swiftyy-mage** [comment]
Something along these lines? To be honest I was a little unsure as to what I should actually test. I was thinking originally I would use `load_playback_file` in `conftest.py`, prepare the data, call `apply_descrambler` on the data, and compare the resulting `player_config_args` with what I knew to be correct. However `player_config_args` is large and I wanted to keep the test neat. Let me know what you think anyway, I can change whatever.

File: pytube/extract.py (hunk 1 above)
Comment (swiftyy-mage) [reply pytube/extract.py]: Done this now.

**Comment by halesyy** [comment]
This is a big issue for me, as I work a lot with long-form HQ videos. Thanks to anyone working on this.

**Pull request merged**